acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/ValidatorSlashingDetectionAcceptanceTest.java

```diff
@@ -0,0 +1,474 @@
+    final UInt64 altairEpoch = UInt64.valueOf(100);
+
+    final TekuNode beaconNode =
+        createTekuNode(
+            config -> configureNode(config, genesisTime, network).withAltairEpoch(altairEpoch));
+
+    final TekuValidatorNode validatorClient =
+        createValidatorNode(
+            config ->
+                config
+                    .withNetwork("auto")
+                    .withStopVcWhenValidatorSlashedEnabled()
+                    .withInteropValidators(0, 32)
+                    .withBeaconNode(beaconNode));
+
+    beaconNode.start();
+    validatorClient.start();
+
+    beaconNode.waitForEpochAtOrAbove(2);
+
+    final int slashedValidatorIndex = 3;
+    final BLSKeyPair slashedValidatorKeyPair = getBlsKeyPair(slashedValidatorIndex);
+
+    postSlashing(
+        UInt64.valueOf(3),
+        UInt64.valueOf(slashedValidatorIndex),
+        slashedValidatorKeyPair.getSecretKey(),
+        beaconNode,
+        slashingEventType);
+
+    validatorClient.waitForLogMessageContaining(
+        String.format(slashingActionLog, slashedValidatorKeyPair.getPublicKey().toHexString()));
+  }
+
+  @ParameterizedTest
+  @MethodSource("getSlashingEventTypes")
+  void shouldShutDownWhenOwnedValidatorSlashed_SingleProcess_MultiplePeers(
+      final SlashingEventType slashingEventType) throws Exception {
+
+    final int genesisTime = timeProvider.getTimeInSeconds().plus(10).intValue();
+    final UInt64 altairEpoch = UInt64.valueOf(100);
+
+    final TekuNode firstTekuNode =
+        createTekuNode(
+            config ->
+                configureNode(config, genesisTime, network)
+                    .withRealNetwork()
+                    .withAltairEpoch(altairEpoch)
+                    .withInteropValidators(0, 32));
+
+    firstTekuNode.start();
+
+    firstTekuNode.waitForEpochAtOrAbove(1);
+
+    final TekuNode secondTekuNode =
+        createTekuNode(
+            config ->
+                configureNode(config, genesisTime, network)
+                    .withAltairEpoch(altairEpoch)
+                    .withStopVcWhenValidatorSlashedEnabled()
+                    .withInteropValidators(32, 32)
+                    .withPeers(firstTekuNode));
+
+    secondTekuNode.start();
+
+    firstTekuNode.waitForEpochAtOrAbove(2);
+
+    final int slashedValidatorIndex = 34;
+    final BLSKeyPair slashedValidatorKeyPair = getBlsKeyPair(slashedValidatorIndex);
+
+    postSlashing(
+        UInt64.valueOf(3),
+        UInt64.valueOf(slashedValidatorIndex),
+        slashedValidatorKeyPair.getSecretKey(),
+        firstTekuNode,
+        slashingEventType);
+
+    secondTekuNode.waitForLogMessageContaining(
+        String.format(slashingActionLog, slashedValidatorKeyPair.getPublicKey().toHexString()));
+
+    firstTekuNode.stop();
+  }
+
+  @ParameterizedTest
+  @MethodSource("getSlashingEventTypes")
+  void
+      shouldShutDownWhenOwnedValidatorSlashed_SingleProcess_MultiplePeers_SlashingEventsThroughGossipOnly_NoBlocks(
```

Comment:
I don't understand why it's `NoBlocks` case, both nodes will have all blocks in this case. Protection activation could be the result of either gossip or block in this case.

Reply:
Same here. In the "no blocks" scenarios we're simply not running validators and assuming that node won't produce any block...